src/makecode/generate-custom-scripts.ts

```diff
@@ -48,8 +48,8 @@ const arrayBufferToHexString = (input: Uint8Array): string =>
 
 export const getAutogeneratedTs = (gs: GestureData[], m: LayersModel) => {
   const customHeaderBlob = generateBlob({
-    samples_period: 25,
-    samples_length: 80,
+    samples_period: 20, // Default value for accelerometer period.
+    samples_length: 50, // Number of samples required at 20 ms intervals for 1 second of data.
```

Comment:
We think this needs to be parameterised for legacy projects.

Reply:
Done here, a bit more change than  I hoped [dededc1](https://github.com/microbit-foundation/ml-trainer/pull/431/commits/dededc1a2265f9fff4215c5539287a1cba8e558d)